cmd/tester/main.go

```diff
@@ -6,27 +6,56 @@ package main
 
 import (
 	"fmt"
+	"log"
+	"os"
+	"strings"
 	"time"
 
 	"github.com/openfaas-incubator/connector-sdk/types"
 )
 
 func main() {
-	creds := types.GetCredentials()
-	config := &types.ControllerConfig{
-		RebuildInterval: time.Millisecond * 1000,
-		GatewayURL:      "http://127.0.0.1:8080",
-		PrintResponse:   true,
-	}
+	creds := types.GetCredentials() // Get credentials for gateway login
+	config := getControllerConfig()
 
 	controller := types.NewController(creds, config)
 	fmt.Println(controller)
 	controller.BeginMapBuilder()
 
+	topics := getTopics()
+
 	// Simulate events emitting from queue/pub-sub
 	for {
 		time.Sleep(2 * time.Second)
 		data := []byte("test " + time.Now().String())
-		controller.Invoke("test", &data)
+
+		for _, topic := range topics {
+			controller.Invoke(topic, &data)
+		}
+	}
+}
+
+func getControllerConfig() *types.ControllerConfig {
```

Comment:
If running in a the default is `gateway.openfaas:8080` / `gateway:8080` on Swarm. We should just not default at all if not passed and return an error `(types.ControllerConfig, error)`